### PR TITLE
Adding PMD linter option

### PR DESCRIPTION
**Pull request by apallin**

### CODE REVIEW

File: linty_fresh/main.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import argparse
 import asyncio
 
-from linty_fresh.linters import android, checkstyle, mypy, pylint, swiftlint
+from linty_fresh.linters import android
```
Comment (TimelyToga) [inline linty_fresh/main.py]: Nit, but was this change intentional?

**Comment by TimelyToga** [comment]
+1

File: linty_fresh/main.py (hunk 1 above)
Comment (apallin) [inline linty_fresh/main.py]: yeah, the line was too long for the linter so I split it to single line imports.  Its actually slightly preferred but you can do it either way.

File: tests/linters/test_pmd.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    </file>
+</pmd>
+    """.format(os.path.curdir)
+
+            result = pmd.parse(test_string)
+            self.assertEqual(2, len(result))
+            self.assertIn(Problem('./DriverRideDrivingToWaypointController.java',
+                                  287,
+                                  'UnusedFormalParameter: Avoid unused method' 
+                                  ' parameters such as \'ride\'.'),
```
Comment (rowillia) [inline tests/linters/test_pmd.py]: It's OK to use double quoted strings around string containing single quotes.

**Comment by rowillia** [comment]
+1
